brokkr/brokkr.mk

```diff
@@ -23,7 +23,7 @@ $(_BROKKR_PLUGINS_SENTINEL):
 $(_BROKKR_PLUGINS_MK): .SHELLFLAGS := -c
 $(_BROKKR_PLUGINS_MK): $(_BROKKR_PLUGINS_SENTINEL)
 	@# Clean working dir, except sentinel
-	@find $(_BROKKR_PLUGINS_DIR) -type f -not -name "`basename $(_BROKKR_PLUGINS_SENTINEL)`" | xargs rm
+	@find $(_BROKKR_PLUGINS_DIR) -type f -not -name "`basename $(_BROKKR_PLUGINS_SENTINEL)`" | xargs -r rm
```

Comment:
Should the `-r` be at the end, like `xargs rm -r`?

Reply:
I think so, will fix

Reply:
Actually not https://stackoverflow.com/a/36618260

Reply:
Because it's an option to xargs not rm

Reply:
Hmmm...
```
mkdir -p /tmp/jass && echo /tmp/jass | xargs -r rm
rm: /tmp/jass: is a directory
```

Reply:
While `mkdir -p /tmp/jass && echo /tmp/jass | xargs rm -r` worked

Reply:
I misunderstood. You are only deleting files anyways.

Reply:
Ok, maybe we need it to be `xargs -r rm -r` then

Reply:
Yes, I guess that's what the –type f in the find is for